.github/workflows/create-github-release.yml

```diff
@@ -0,0 +1,53 @@
+name: Create GitHub release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        type: string
+        description: 'Version to publish'
+        required: true
+      title:
+        type: string
+        description: 'Release title'
+        required: true
+      deploy-sha:
+        type: string
+        description: 'Commit SHA to create release from'
+        required: true
+  
+  workflow_call:
+    inputs:
+      version:
+        type: string
+        description: 'Version to publish'
+        required: true
+      title:
+        type: string
+        description: 'Release title'
+        required: true
+      deploy-sha:
+        type: string
+        description: 'Commit SHA to create release from'
+        required: true
+
+jobs:
+  create-github-release:
+    name: Create GitHub release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create release
+        uses: ncipollo/release-action@v1
+        with:
+          name: ${{ inputs.title }}
+          tag: ${{ inputs.version }}
+          token: ${{ github.token }}
+          generateReleaseNotes: true
+          commit: ${{ inputs.deploy-sha}}
+      - name: Notify Slack about successful release
+        uses: ravsamhq/notify-slack-action@v2
+        with:
+          notification_title: "New constructor-ui-autocomplete GitHub release created at version ${{ inputs.version }}"
```

Comment:
**Small suggestion:** Every other notification has emojis, should we add one here too?

```suggestion
          notification_title: "😸 New constructor-ui-autocomplete GitHub release created at version ${{ inputs.version }}"
```

Reply:
Added in the latest commit!